reviews.Rmd

```diff
@@ -0,0 +1,583 @@
+:::
+
+We agree that this argument could perhaps be made for a frequentist as well.
+However, it is not the math of the likelihood ratio that eludes us; it is how a significant result of a statistical test changes how a frequentist judges a hypothesis.
+There is a great, yet unresolved, debate in frequentism about how to do just this.
+In a Fisherian approach, no heat is paid to the power of a test; only the p-value is interpreted as the strength of evidence against the null.
+In the Neyman-Person paradigm, a decision is to be made about rejecting one or the other hypothesis.
+Mayo's severe testing returns to a gradual inference (instead of a decision) by evaluating the extent of evidence in favour of H as the extent to which a test was capable of showing that H is false.
+The Bayesian position deals with what agents (here "researchers") ought to believe, assuming they are rational.
+However, even here, there is no universally agreed upon function that maps evidence onto belief (indeed, there can not be).
+We tried to make this even clearer and more explicit as to why we include several functions (and that our reasoning applies to all possible functions satisfying our assumptions).
+The frequentist understanding of probability refers strictly to statistical procedures.
+From this position, one must either argue that preregistration is changing the underlying random process/ statistical procedure or trace the impact of preregistration via indirect paths outside of the core philosophy of frequentism.
+Mayo's severity perspective is taking one more step in this direction by asserting that belief in a hypothesis is justified to the extent that it survives stringent scrutiny.
+However, even this "strong" severity principle is only a proportional statement.
+Faced with a specific tradeoff, e.g., is a particular post hoc change of a preregistration beneficial, we must revert to vague notions and "gut" instinct under this philosophy.
+Within the Bayesian framework, we may directly ask: What consequences should preregistration have on the beliefs of individual scientists or the research community as a whole?
+
+:::reviewer
+The title should not read a Bayesian rationalization, but a likelihood rationalization.
+A higher likelihood is not always better, as one can cherry pick a high likelihood ratio.
+This is extensively discussed by Mayo (2018).
+After p-hacking, we could get a very high likelihood (or Bayes factor) that supports H1 - but it was cherry-picked, and is highly likely to be a fluke.
+Therefore, the authors limit themselves to relative likelihoods.
+:::
+
+We did not limit ourselves to relative liklihoods.
+We explicitly include both the "firmness" and "increase in firmness" perspective.
+That a firmness perspective has significant flaws is of no matter to our argument since we intend to show that someone who subscribes to either must be more persuaded by a preregistered study.
```

Comment:
```suggestion
We explicitly include both the "firmness" and "increase in firmness" perspective.
Since we showed that out justification of preregistration holds for both perspectives, the discussion about specific flaws of the firmness perspective do not concern our argument.  
```

Reply:
added